prow/jobs/generated/knative/serving-main.gen.yaml

```diff
@@ -683,7 +683,7 @@ periodics:
       - name: SYSTEM_NAMESPACE
         value: knative-serving
       - name: TEST_OPTIONS
-        value: --enable-alpha --enable-beta --resolvabledomain=false
+        value: --enable-alpha --enable-beta --resolvabledomain=false --short
```

Comment:
The `-short` doc: https://github.com/golang/go/blob/d11c58eedbee29b4912dd50508b36ad15ebb739e/src/cmd/go/internal/test/test.go#L311-L315

I think the `-short` flag should be used for unit test.

It is also used in default unit test runner in Knative: https://github.com/knative/hack/blob/d1317b08c0c3a7180e1dd9e9faec7532fdfe615f/presubmit-tests.sh#L205

Reply:
Better would be to introduce a clear, flake related flag/envvar like: `-skip-flakes`

Reply:
Well, it would seem that knative is using the '--short' flag differently to what golang proposes - at least in context of the e2e tests. If you look at the current serving test cases you'll find two occurrences of `testing.Short()` which are used to disable specific scale tests, one of them directly noting that KinD clusters are insufficient for running those tests. My goal with this PR is to piggyback on existing integrations, not coming up with new ways of how to do things.

Reply:
I clearly get your intention of piggybacking. But, I think that's wrong. We shouldn't put flaky tests to slow bucket. That's why I'm proposing to introduce flake-specific flag instead.

Reply:
Thanks Chris for your input. I'll think of something else. For now I will close this PR.